app/routes.test.js

```diff
@@ -177,6 +181,92 @@ describe('routes', () => {
     });
   });
 
+  describe('POST /organisations/:organisationId', () => {
+    it('should return 403 forbidden if no csrf token is available', () => request(setUpFakeApp())
+      .post('/organisations/org1')
+      .set('Cookie', [mockAuthorisedCookie])
+      .type('form')
+      .send({})
+      .expect(403));
```

Comment:
You are gonna hate me 😭 but we have a few of these now. They are pretty much identical for the POST routes behaviour we are testing. Can we have a little helper functions like we do for the `GET checkAuthorisedRouteNotLoggedIn` that do this that we can re-use for all the post endpoints.

Reply:
this was your job 😄

Reply:
Do you want me to take a look at it? I don't mind